src/style.css

```diff
@@ -1 +1,83 @@
 /* add styles here */
+@font-face {
+  font-family: 'Avenir normal';
+  src: url(/src/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Avenir bold';
+  src: url(/src/fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: bold;
```

Comment:
The `font-weight` for 'Avenir bold' is set to 900, which might not align with the task requirements. Please ensure it matches the specified requirements.